py/mattak/backends/uproot/dataset.py

```diff
@@ -244,7 +255,67 @@ def eventInfo(self) -> Union[Optional[mattak.Dataset.EventInfo],Sequence[Optiona
     
     def N(self) -> int: 
         return self._hds.num_entries
+    
+    def __unpack_cal_parameters(self, cal_file : uproot.ReadOnlyDirectory) -> numpy.ndarray:
+        """
+        Function that reads out 9th order polynomial parameters from the calibration file
+
+        Returns
+        ---------
+        coef : np.ndarray of shape (24 * 4096, 10)
+        """
+        coef = numpy.stack(cal_file["coeffs_tree/coeff"].array(library = 'np'))   #stack is needed to convert ndarray of ndarrays to 'normally shaped' array, otherwise you have nested ndarrays
+        return coef
+
+    def __unpack_cal_residuals(self, cal_file : uproot.ReadOnlyDirectory) -> numpy.ndarray:
+        """
+        Function that reads out the residuals from the calibration file
+
+        Returns
+        ---------
+        (v_residuals, residuals) : both v_residuals and residuals have shape (points, 2)
+
+        """
+
+        vres_dac1, vres_dac2 = cal_file["aveResid_dac1"].values(axis = 0), cal_file["aveResid_dac2"].values(axis = 0)
```

Comment:
can you split that in two lines? I think that is more in agreement with our coding style

Reply:
And all occasions where two variables are initialised in a single line.